sos/report/plugins/__init__.py

```diff
@@ -1912,6 +1922,16 @@ def _collect_cmd_output(self, cmd, suggest_filename=None,
         _tags.append(cmd.split(' ')[0])
         _tags.extend(self.get_tags_for_cmd(cmd))
 
+        if cmd_as_tag:
+            _cmdtag = cmd
+            for s in (' ', '/', '-'):
+                _cmdtag = _cmdtag.replace(s, '_')
+            # reduce duplicates to a single underscore
+            _cmdtag = _cmdtag.replace('__', '_')
```

Comment:
Can't we use `_mangle_command` from https://github.com/sosreport/sos/blob/master/sos/report/plugins/__init__.py#L36, as being used for `outfn` in https://github.com/sosreport/sos/blob/master/sos/report/plugins/__init__.py#L1803 ? Then tag would match the `outfn` filename.

Reply:
Hadn't thought of that, but yeah that makes sense.